RecoParticleFlow/PFProducer/python/HGCalTrackCollection_cfi.py

```diff
@@ -0,0 +1,19 @@
+import FWCore.ParameterSet.Config as cms
+
+HGCalTrackCollection = cms.EDProducer(
```

Comment:
name module instances with lower case
.. and keep the file name and the instance module name the same